### ISSUE TITLE
[FR] Create opacity styles without color or other attributes.

### ISSUE THREAD

**Issue by mikebronner** [ingrammicro/puzzle-tokens#10]
It is beneficial to create styles that only contain opacity settings without any other attributes. This would allow us to apply them to groups and other containers as a whole.

I attempted this, but the plugin doesn't seem to create the styles correctly. I tried both decimal and percentage values.
```less
.Opacities .0 {
  opacity: 0;
}
.Opacities .25 {
  opacity: 0.25;
}
.Opacities .50 {
  opacity: 0.5;
}
.Opacities .75 {
  opacity: 0.75;
}
.Opacities .100 {
  opacity: 1;
}
```

**Comment by MaxBazarov** [comment]
Interesting idea. I need to discover it.
Currently you can apply opacity to layer/group directly, without style creation. See example here - 
https://github.com/ingrammicro/puzzle-tokens/tree/master/Styles/Tests/Opacity

**Comment by mikebronner** [comment]
Thanks. Yea, I saw that, but that predicates that the symbols already exist. I creating a design system, where the designer should be able to apply opacity where needed, and select from a preset list of opacity options. :)

**Comment by MaxBazarov** [comment]
Ok, I understand. Will implement it tomorrow.

**Comment by MaxBazarov** [comment]
Implemented in 6.2.0
